h264/bits/bitreader_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+DESCRIPTION
+  bitreader_test.go provides testing for functionality defined in bitreader.go.
+
+AUTHORS
+  Saxon Nelson-Milton <saxon@ausocean.org>, The Australian Ocean Laboratory (AusOcean)
+*/
+
+package bits
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/pkg/errors"
```

Comment:
There are no wrapped errors in the package. This is not necessary.

Reply:
I'm using errors.Cause() though

Reply:
Yes, I know. There is not reason for that.

Reply:
yeah, not anymore